### PR TITLE
Feature/soroban fund account

### PR DESCRIPTION
**Pull request by AndresNazzari**
# Summary

This PR adds a node for `fundAccount` in the Soroban platform through n8n.

# Details

- Add fundAccount action
- Add fundAccount action to the Soroban node
- Add soroban rcp futurenet network

# Evidence

image
image
image

# References

📖 [JS Soroban Client](https://stellar.github.io/js-soroban-client/)

### CODE REVIEW

File: nodes/Soroban/actions/fundAccount/fundAccount/execute.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import StellarSdk from 'stellar-sdk';
```
Comment (Genebson) [inline nodes/Soroban/actions/fundAccount/fundAccount/execute.ts:1]: Do not import the whole dependency, import just what we need, please.

File: nodes/Soroban/actions/fundAccount/fundAccount/execute.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import type { IExecuteFunctions, IHttpRequestOptions } from 'n8n-workflow';
+import InvalidPublicKeyError from '../../../../../common/errors/stellar/InvalidPublicKeyError';
+
+const FRIENDBOT_URL = 'https://friendbot-futurenet.stellar.org/?addr=';
+
+export async function fundAccount(this: IExecuteFunctions) {
+	try {
+		const items = this.getInputData();
+		const publicKey = items[0].json.publicKey as string;
```
Comment (Genebson) [inline nodes/Soroban/actions/fundAccount/fundAccount/execute.ts:10]: I think we can destructure publicKey here.

Comment (AndresNazzari) [reply nodes/Soroban/actions/fundAccount/fundAccount/execute.ts:10]: fixed

File: nodes/Soroban/actions/fundAccount/fundAccount/execute.ts (hunk 1 above)
Comment (AndresNazzari) [reply nodes/Soroban/actions/fundAccount/fundAccount/execute.ts:1]: fixed

**Review by Genebson — APPROVED** [review]

**Pull request merged**